docs/music.rst

```diff
@@ -92,14 +95,20 @@ Functions
     their defaults before the music (whatever it may be) is played.
 
     An optional argument to specify the output pin can be used to override the
-    default of ``microbit.pin0``.
+    default of ``microbit.pin0``. If you have the latest micro:bit, you can
+    use ``microbit.pin_speaker`` **V2**.
+
+    The pin argument can also take a tuple of two pins, for example
```

Comment:
Joining these two sentences, otherwise it sounds like you can use a tuple in V1 as well.


```suggestion
    default of ``microbit.pin0``. With micro:bit **V2** you can use
    ``microbit.pin_speaker`` or a tuple of two pins, for example
```